core/storage/email/gae_models.py

```diff
@@ -144,15 +144,15 @@ def get_by_hash(cls, email_hash, sent_datetime_lower_bound=None):
 
         Args:
             email_hash: str. The hash value of the email.
-            sent_datetime_lower_bound: Date and time. The lower bound on 
-            sent_datetime of the email to be searched.
+            sent_datetime_lower_bound: datetime.datetime. The lower bound on 
+                sent_datetime of the email to be searched.
 
         Returns:
-            The message searched given the hash value of email.
-
+            A list of messages.The hashed messages with the given hash value
```

Comment:
list([???]). A list of [???] which have the given hash value and which were sent more recently than sent_datetime_lower_bound.

(The [???] are for you to fill in.)